backend/src/services/group/group.service.ts

```diff
@@ -1,8 +1,9 @@
 import { ExceptionMessage, StringCase } from '~/common/enums/enums';
 import {
-  type GroupsGetAllItemResponseDto,
-  type GroupsGetAllResponseDto,
```

Comment:
i think now we need to delete GroupsGetAllItemResponseDto & GroupsGetAllResponseDto